client/components/NewMessage.jsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react'
+
+import people from '../people.js'
+
+const styles = {
+  borderRadius: '150px',
+  background: 'white',
+  width: '300px',
+  height: '300px'
+}
+
+class NewMessage extends React.Component {
+  constructor () {
+    super()
+    this.state = {
+      sender: null,
+      recipient: null,
+      senderPhoto: null,
+      recipientPhoto: null
+    }
+    this.handleChange = this.handleChange.bind(this)
+    this.handleGetPhoto = this.handleGetPhoto.bind(this)
+  }
+  handleGetPhoto (action, name) {
+    if (action === 'sender') {
+      people.map((person) => {
```

Comment:
Since you're not using the array returned from `.map`, you might do better to use `.find`.